doc/releases.rst

```diff
@@ -12,27 +12,45 @@ Releases
 Version 0.15.0
 ==============
 
-* ADD #1317: Initial support for multi-objective Auto-sklearn.
+* ADD #1317, #1455, #1485, #1501, #1518, #1523: Initial support for multi-objective Auto-sklearn.
+* ADD #1300, #1410, #1414, #1415, #1420, #1468, #1500: Intial support for text features Auto-sklearn. You can now pass in columns identified as `"string"` columns which will be tokenized using pure sklearn methods.
+* ADD #1475: Support for passing `X` data to metrics, as required by [`fairlearn`](https://github.com/fairlearn/fairlearn) metrics
+* ADD #1341, #1250: Expose interface to interact with how auto-sklearn performs dataset compression when required
+
 * DOC #1304: This adds documentation for SMAC callbacks that can be used by Auto-sklearn.
-* MAINT #1364: Improve import time of Auto-sklearn 2 by moving the construction of the selector
+* DOC #1476: Example on how to interupt autosklearn with a callback, implementing a very naive early stopping
+
+* MAINT #1364, #1473: Improve import time of Auto-sklearn 2 by moving the construction of the selector
   model from import time to construction time.
+* MAINT #1425: Update `StopWatch` to be context manager.
+* MAINT #1454: Rename interal bool parameters `categorical` to `feat_type` to reflect the use of different feature types
 * MAINT #1474: remove left-overs of a "public test set" from the code. This has no influence on
   any user-facing code.
+* MAINT #1487: Replace deprecated of `DataFrame.append`
+* MAINT #1504: Rename `rval` to `return_value` or `run_value` to remove ambiguity
 * MAINT #1506: Increase the time given to meta-learning-related unit tests to decrease the amount
   of timeouts on github.
 * MAINT #1527: Relax MLPRegressor unit tests precision.
+* MAINT #1545: Add explicit lower bound subsample check in the train evaluator
+* MAINT #1551: Fix issue with updated scipy skew see [here](https://github.com/scipy/scipy/issues/16765).
+* MAINT #1434: Refactor the ensemble building process
+* MAINT #1464: Improve testing, with caching (#1464), modularity (#1417)
+* MAINT #1358: Add tooling Mypy, Flake8, isort, black
+
 * FIX #741: Disable hyperparameters for a special data modality if it is not present, for example
   disable one hot encoding if no categorical features are present.
-* FIX #1365: Fix an issue with `ensemble_size == 0`.
+* FIX #1365, #1369: Fix an issue with `ensemble_size == 0`.
 * FIX #1374: Pass random state to all components of a pipeline.
 * FIX #1430: Fixes an issue in which the `AutoSklearnClassifier.leaderboard()` or
   `AutoSklearnRegressor.leaderboard()` could fail to display results.
+* FIX #1432: Fill the empty leaderboard slots with NaN.
```

Comment:
Isn't this the same one as above @eddiebergman ?

Reply:
Yup it is, can we stick to generating the release notes from commits to dev in the future, it's more comprehensive than just closed issues and prevents this issue occuring where we have both the issue number and the PR number but they highlight the same thing.

Reply:
Makes sense. And thanks for fixing this nevertheless. Now that we're converged here, let's merge this one and prepare the larger merge.